cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -29,8 +29,10 @@ class DemuxPostProcessingAPI:
 
     def __init__(self, config: CGConfig) -> None:
         self.config: CGConfig = config
-        self.flow_cells_dir: Path = Path(config.illumina_flow_cells_directory)
-        self.demultiplexed_runs_dir: Path = Path(config.illumina_demultiplexed_runs_directory)
+        self.flow_cells_dir: Path = Path(config.run_instruments.illumina.flow_cell_runs_dir)
```

Comment:
new config paths that we use for run instruments, this already exists in config and servers